internal/provider/resource_backbone.go

```diff
@@ -170,6 +172,34 @@ func resourceBackbone() *schema.Resource {
 				},
 			},
 		},
+		CustomizeDiff: customdiff.Sequence(
+			func(_ context.Context, d *schema.ResourceDiff, m interface{}) error {
```

Comment:
It is worth considering starting to add unit tests for these functions.

Reply:
I will create an issue in the backlog. We can do that once the current project regarding acceptance test is completed.